src/status_im/ui/components/chat_icon/styles.cljs

```diff
@@ -257,7 +256,5 @@
    :justify-content :center})
 
 (def profile-icon-edit-text
-  (merge (get-in p/platform-specific [:component-styles :profile :profile-icon-edit-text])
-         {:color            :white
-          :background-color :transparent}))
-
+  {:color            :white
```

Comment:
Merge is gone?

Reply:
The platform-specific style does not exist at all.

Reply:
👍